translations/strings.yaml

```diff
@@ -316,6 +299,25 @@ settings:
   crashOptInDescription:
     value: Allow Mozilla VPN to automatically send crash reports to help fix unexpected crashes
     comment: This is the description of a checkbox option to allow the app to send crash reports without a user prompt
+  privacyAdblockTitle: Block ads
+  privacyAdblockBody: Reduces the number of ads you see when you’re using VPN
+  privacyTrackerTitle: Block trackers
+  privacyTrackerBody: Fewer harmful domains will be able to track you
+  privacyMalwareTitle: Block malware
+  privacyMalwareBody: Helps protect you from ransomware, phishing, spyware, viruses, and malware
+  dnsSettingsStandardDNSTitle: Use standard DNS
+  dnsSettingsStandardDNSBody: Resolve websites using Mozilla VPN’s standard DNS
+  dnsSettingsCustomDNSTitle: Use custom DNS
+  dnsSettingsCustomDNSBody: Resolve websites using a custom DNS
+  dnsSettingsCustomDNSError: Invalid DNS, please try a new one
+
+dnsOverwriteDialog:
+  titleDNS: Override DNS settings?
+  titlePrivacy: Override privacy features?
+  bodyDNS: Using these privacy features will override your custom DNS.
+  bodyPrivacy: Using a custom DNS will override your preferences in the privacy features page.
```

Comment:
Should this use `Privacy features` with uppercase, since it's the actual title of the page?

Reply:
@chrisrooneys ^